src/app/launcher/create-app/gitprovider-createapp-step/gitprovider-createapp-step.component.ts

```diff
@@ -115,9 +115,13 @@ export class GitproviderCreateappStepComponent extends LauncherStep implements A
   validateRepo(): void {
     let fullName = this.launcherComponent.summary.gitHubDetails.organization + '/'
       + this.launcherComponent.summary.gitHubDetails.repository;
+    let org = this.launcherComponent.summary.gitHubDetails.organization;
+    let repoName = this.launcherComponent.summary.gitHubDetails.repository;
 
-    this.subscriptions.push(this.gitProviderService.isGitHubRepo(fullName).subscribe((val) => {
-      if (val !== undefined) {
+    this.subscriptions.push(this.gitProviderService.isGitHubRepo(org, repoName).subscribe((val) => {
+      if (val) {
+        this.launcherComponent.summary.gitHubDetails.repositoryAvailable = !val;
+      } else {
         this.launcherComponent.summary.gitHubDetails.repositoryAvailable = !val;
         this.initCompleted();
```

Comment:
There should not be an 'else' statement here. We just need to set the value and call initCompleted when the summary has changed.

The initCompleted call is needed in order to show the completed steps properly.

Reply:
During testing, I clicked on the enabled "Set up Application" button in the summary page. However, I never get to the progress screen.

I suspect this is related to the above change, but not certain?

Reply:
@dlabrecq where you testing sourcing with prod-preview or stand alone? as in stand alone you will get 401. In case of sourcing with prod-preview will make changes to prompt for now in case of any validation err message

Reply:
I'm just running the demo right now. Are you npm linking to fabric8-ui? That's fine, I just need to understand how to test it. 

I know you plan to create a PR for fabric8-ui to introduce changes there. I assume that will also be needed to test properly?

Reply:
Thanks Dan. This PR doesn't have git APIs , it has all other APIs. While introducing git flow i'll make sure i test it with fabric-ui

Reply:
@dlabrecq Currently I assume, we don't have the facility to show error messages in the getting started flow. You can follow this discussion here for the same https://github.com/fabric8-launcher/launcher-planning/issues/125